.github/actions/compile-make/action.yml

```diff
@@ -0,0 +1,37 @@
+name: Compile make
+description: compile-make
+inputs:
+  version:
+    description: make version
+    required: true
+  workaround:
+    description: enable workaround for _alloc bug
+    required: false
+    default: "false"
+
+runs:
+  using: composite
+  steps:
+    - name: Cache make compiled
+      if: ${{ !startsWith(runner.os, 'windows') }}
+      id: cache-maka
+      uses: actions/cache@v4
+      with:
+        path: /usr/local/bin/make-${{ inputs.version }}
+        key: v1-${{ runner.os }}-make-${{ inputs.version }}
+
+    # Compile it from source (temporarily)
```

Comment:
Temporarily? Is this is meant to be a FIXME? If yes can you link to an issue or comment?

Reply:
Well, this is a comment I copied from `main.yml`

It was there because `ubuntu-latest` uses make <4.3 yet we want to test with make 4.4

Reply:
I've removed the confusing comment